src/Sentry/Android/Sentry.Android.props

```diff
@@ -5,7 +5,7 @@
     <!-- BG8605 and BG8606 happen because there's a missing androidx.lifecycle dependency, but we don't need it here.  (The native Android Sentry SDK will use it if it exists.) -->
     <NoWarn>$(NoWarn);BG8605;BG8606</NoWarn>
     <SentryAndroidSdkVersion>6.0.0</SentryAndroidSdkVersion>
-    <AndroidLibsDirectory>$(BaseIntermediateOutputPath)\libs\Android\</AndroidLibsDirectory>
+    <AndroidLibsDirectory>$(BaseIntermediateOutputPath)\SentrySDKs\Android\$(SentryAndroidSdkVersion)\</AndroidLibsDirectory>
```

Comment:
Less relevant on Android because artifacts have the version on the file name, but puts things under the version to avoid issues when we update:

![image](https://user-images.githubusercontent.com/1633368/174439478-7f8fd78a-64bb-4c61-a4ab-24a7b2982ac0.png)